arch/arm/boot/dts/overlays/revpi-flat-s-2022-overlay.dts

```diff
@@ -0,0 +1,367 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Flat S (2022)
+ */
+
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/{
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-flat-s-2022",
+				     "kunbus,revpi-flat", "brcm,bcm2711";
+
+			aout_vref: fixedregulator_2v5x4 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x4_amplified";
+				regulator-min-microvolt = <10000000>;
+				regulator-max-microvolt = <10000000>;
+				regulator-always-on;
+			};
+
+			ain_vref: fixedregulator_2v5x5 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x5_amplified";
+				regulator-min-microvolt = <12500000>;
+				regulator-max-microvolt = <12500000>;
+				regulator-always-on;
+			};
+
+			wl_on: reg_wl {
+				compatible = "mmc-pwrseq-simple";
+				reset-gpios = <&expander 10 GPIO_ACTIVE_LOW>;
```

Comment:
I would use the `WLAN_BT_PDn` for the regulator.

Reply:
> There is also the question about the definition of the pinmuxes. You still reference the definitions from the cm4s dtsi. Do we want to keep that?

I thought I overwrote all references. Which one do you mean still exist?